### ISSUE TITLE
Any chance of getting the description back?

### ISSUE THREAD

**Issue by Gameniti** [2004content/rarbg#15]
Hi there,
first of all thanks for your project.

There was a description for every file which (sometimes) stated the included languages (dub and sub).
Is there any way to get these back so you can search/find files that are not English only?

Thanks

**Comment by 2004content** [comment]
No, I really can only handle the indexing of the magnets; the size of the
files and the processing complexity to include anything else is just not
really possible. It's also not really necessary for its intended purpose. If
you're talking about the descriptions from RARBG's site, I don't think
anyone has those anyway.  What I can tell you though, is that any of the
magnets marked -RARBG (probably almost/around 200k if I had to guess off
the top of my head) have full sub folders once downloaded. I think a lot of
the higher quality -FGTs and stuff have subs too, but I'm not as familiar
with those so maybe someone else can confirm.

**Comment by Gameniti** [comment]
Yeah I was talking about the description on the site.
Most of the time it said which audio tracks were included since I mostly want them in my language.
Thanks anyway for your response and your project.
